### DISCUSSION TITLE
Reporting errors in Artsdata

### DISCUSSION THREAD

**Discussion by dlh28** [culturecreates/artsdata-data-model#132] [General] [answered]
At last week's Digital Team meeting, @fjjulien, Gary, Morgan and I were discussing how to report errors that we notice in Artsdata (I was looking specifically at [Théâtre Le Mack](https://kg.artsdata.ca/entity.html?uri=http%3A%2F%2Fkg.artsdata.ca%2Fresource%2FK4-122), which has a statement “containedinPlace: Confederation Centre”, although it is actually a standalone building operated by the Confederation Centre – I suspect this error came from Wikidata, where I have since fixed the misclassification).

@saumier would you prefer if we posted them as individual issues in GitHub and tagged you? Or would it be better to compile them in a spreadsheet and email them to you at a regular interval (i.e. once a month or so)?

**Comment by saumier** [comment, marked as answer]
Lets use issues in this repo https://github.com/artsdata-stewards/support-issues/issues
